examples/strands/strands.md

```diff
@@ -2,7 +2,7 @@
 
 By default, named workers are multitasked cooperatively, not preemptively. Each named worker has a "strand" (logical thread of control) and execution switches between strands only at specific "yield" points such as doing a wait or when a library function invokes a system call that would block.
 
-This avoids the need for users to lock variables that are accessed from multiple named workers. An annotation can be used to make a strand run on a separate thread.
+This avoids users' need to lock variables accessed from multiple named workers.
```

Comment:
We can remove the `An annotation can be used to make a strand run on a separate thread.` part since the `@strand` annotation is deprecated right?

Reply:
+1 to have it like before.
``` suggestion
This avoids the need for users to lock variables that are accessed from multiple named workers.
```

Reply:
> We can remove the `An annotation can be used to make a strand run on a separate thread.` part since the `@strand` annotation is deprecated right?

Yes.

Reply:
Restored with https://github.com/ballerina-platform/ballerina-distribution/pull/5288/commits/bf13bc42e02c52975d2e5d4b0d974911f76726bd